pysat/utils/io.py

```diff
@@ -172,121 +167,540 @@ def filter_netcdf4_metadata(inst, mdata_dict, coltype, remove=False,
+                else:
+                    name = var
+                update_dict = {inst.meta.labels.name: name}
+                update_dict.update(meta_dict)
+
+            if lower_var in out_meta_dict:
+                out_meta_dict[lower_var].update(update_dict)
+            else:
+                warnings.warn(''.join(['Unable to find MetaData for ',
+                                       var]))
+                out_meta_dict[lower_var] = update_dict
+
+            # Filter metdata for other netCDF4 requirements
+            remove = True if index_type == str else False
+            out_meta_dict[lower_var] = filter_netcdf4_metadata(
+                inst, out_meta_dict[lower_var], index_type, remove=remove,
+                check_type=check_type, export_nan=export_nan, varname=lower_var)
+
+        else:
+            # Dealing with 1D data or xarray format
+            meta_dict['Format'] = inst._get_var_type_code(coltype)
+
+            if not inst.pandas_format:
+                for i, dim in enumerate(list(inst[var].dims)):
+                    meta_dict['Depend_{:1d}'.format(i)] = dim
+                num_dims = len(inst[var].dims)
+                if num_dims >= 2:
+                    meta_dict['Display_Type'] = 'Multidimensional'
+
+            # Update the meta data
+            if lower_var in out_meta_dict:
+                out_meta_dict[lower_var].update(meta_dict)
+            else:
+                warnings.warn(''.join(['Unable to find MetaData for ',
+                                       var]))
+                out_meta_dict[lower_var] = meta_dict
+            # out_meta_dict[lower_var].update(meta_dict)
+
+            # Filter metdata for other netCDF4 requirements
+            remove = True if coltype == str else False
+            out_meta_dict[lower_var] = filter_netcdf4_metadata(
+                inst, out_meta_dict[lower_var], coltype, remove=remove,
+                check_type=check_type, export_nan=export_nan, varname=lower_var)
+
+    return out_meta_dict
+
+
+def remove_netcdf4_standards_from_meta(mdict, epoch_name, labels):
+    """Remove metadata variables in SPDF ISTP/IACG NetCDF standards.
+
+    Parameters
+    ----------
+    mdict : dict
+        Contains all of the loaded file's metadata.
     epoch_name : str
-        Name for epoch or time-index variable
+        Name for epoch or time-index variable. Use '' if no epoch variable.
+    labels : Meta.labels
+        `Meta.labels` instance.
+
+    Returns
+    -------
+    mdict : dict
+        File metadata with netCDF4 SPDF information removed.
 
     Note
     ----
+    Removes metadata for `epoch_name`. Also removes metadata such as 'Depend_*',
+    'Display_Type', 'Var_Type', 'Format', 'Time_Scale', 'MonoTon',
+    'calendar', and 'Time_Base'.
+
     Does not perform filtering to remove variables not supported by the
     SPDF ISTP/IACG NetCDF standards.  For this, see
     pysat.utils.io.filter_netcdf4_metadata.
```

Comment:
See previous docstring for how I reworded this phrase last time.

Reply:
I updated the docstring to be more clear (hopefully). I added a See Also but referenced `add_netcdf4_standards_to_metadict` since that is the inverse of this function. The Note itself describes all of the changes to metadata.